src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionQRInfoViewController.swift

```diff
@@ -43,7 +43,7 @@ class ExposureSubmissionQRInfoViewController: DynamicTableViewController {
 	// MARK: - Internal
 
 	enum ReuseIdentifiers: String, TableViewCellReuseIdentifiers {
-		case acknowledgement = "DynamicAcknowledgementCell"
+		case legal = "DynamicAcknowledgementCell"
```

Comment:
```suggestion
		case legal = "DynamicLegalCell"
```
I was wondering why the cell name identifiers stays unchanged